packages/scene-composer/src/components/panels/scene-components/AnimationComponentEditor.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback } from 'react';
```

Comment:
please try to not use `any`. if you run `npm run lint`, you should see it fails due to too many warnings